task/src/main/java/com/flipkart/flux/impl/task/LocalJvmTask.java

```diff
@@ -72,19 +69,28 @@ public int getExecutionTimeout() {
             While this works for methods with all unique param types, it
             will fail for methods where we have mutliple params of the same type.
              */
+            ClassLoader classLoader = toInvoke.getDeploymentUnitClassLoader() != null ? toInvoke.getDeploymentUnitClassLoader() : this.getClass().getClassLoader();
```

Comment:
so this code should never exist. If its the tests that are creating the problem then fix the tests.
Never alter production code just because we're not writing tests correctly :)